### DISCUSSION TITLE
Retrieve the current consumer group id

### DISCUSSION THREAD

**Discussion by abrokenjester** [zendesk/racecar#251] [Q&A] [answered]
For aggregated logging purposes, I am looking to have my racecar consumer report the consumer group id. I am having a hard time figuring out how to get at it though. Is there a way to access this information from the consumer initializer method?

**Comment by dasch** [comment, marked as answer]
As in, you'd like to get it programmatically from within the consumer process? If so, `Racecar.config.group_id` should do the trick.
